### ISSUE TITLE
Bug due to ReLU(inplace=True) with TrainWithClassifier

### ISSUE THREAD

**Issue by alpoler** [KevinMusgrave/powerful-benchmarker#78]

**Comment by KevinMusgrave** [comment]
Where does the error come from? Is it coming from your model or a loss function?

**Comment by alpoler** [comment]
It is coming from MLP layer of classifier because I remove inplace option from there.After that, Problem removes ;however, mlp relu does not contribute backpropagation ( I guess). I don't give mean to why it occurs and problem finishes when I removed inplace from MLP.

**Comment by KevinMusgrave** [comment]
So the bug originates here: https://github.com/KevinMusgrave/powerful-benchmarker/blob/master/src/powerful_benchmarker/architectures/misc_models.py#L76 ?

**Comment by alpoler** [comment]
Exactly. I disable inplace from both embedder and classifier. Problem is solved. Are there any hidden effect due to removal of inplace ? I read that it only effect memory usage.

**Comment by KevinMusgrave** [comment]
Yeah I think it only affects memory usage.
